ansible/roles/openshift_logging/defaults/main.yml

```diff
@@ -1,5 +1,7 @@
 osalog_fluentd_nodes:
 - type=compute
+- type=master
```

Comment:
Shouldn't we have this by default with the openshift-ansible playbook/roles for installing logging?